assets/xml/objects/object_kinsta1_obj.xml

```diff
@@ -1,7 +1,8 @@
 ﻿<Root>
+    <!-- Object for Swamp Spider House Door -->
     <File Name="object_kinsta1_obj" Segment="6">
-        <DList Name="object_kinsta1_obj_DL_000190" Offset="0x190" /> <!-- Original name is "m2_KIN1_shutter_modelT" -->
-        <DList Name="object_kinsta1_obj_DL_000198" Offset="0x198" /> <!-- Original name is "m2_KIN1_shutter_model" -->
-        <Texture Name="object_kinsta1_obj_Tex_0002F0" OutName="tex_0002F0" Format="rgba16" Width="32" Height="64" Offset="0x2F0" />
+        <DList Name="gSwampSpiderHouseDoorEmptyDL" Offset="0x190" /> <!-- Original name is "m2_KIN1_shutter_modelT" -->
+        <DList Name="gSwampSpiderHouseDoorDL" Offset="0x198" /> <!-- Original name is "m2_KIN1_shutter_model" -->
+        <Texture Name="gSwampSpiderHouseTex" OutName="swamp_spider_house_door" Format="rgba16" Width="32" Height="64" Offset="0x2F0" />
```

Comment:
You missed the `Door` in the name, but it should also have `SlidingDoor` in there